examples/websocket-failover/websocket_failover.metatags

```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates how WebSocket failover client connects with the remaining backends when the client looses the connection.
```

Comment:
Edit: This BBE demonstrates how a WebSocket failover client connects with the remaining backends when the client loses the connection.